src/main/resources/application.yml

```diff
@@ -69,11 +69,11 @@ management:
 
 app:
   link:
-    base-url: ${link.base-url:http://localhost:8080/session}
+    base-url: ${link.base-url:http://localhost:3000/form}
   session:
     max-age: ${session.max-age:12}
-  form:
-    url: ${LINK_BASE_URL:http://localhost:3000/form}
+  form-config:
+    base-url: ${form-config.base-url:http://localhost:3000/form}
```

Comment:
If we haven't already, we will need to include this as an environment variable in our Helm values, this should be `FORM_CONFIG_BASE_URL`. We did something similar with `LINK_BASE_URL`

Reply:
Good catch, have added it to the values for each env 👍